common/lib/capa/capa/templates/formulaequationinput.html

```diff
@@ -4,23 +4,14 @@
     <input type="text" name="input_${id}" id="input_${id}"
         data-input-id="${id}" value="${value|h}"
         aria-label="${label}"
+        aria-describedby="${id}_status"
         % if size:
         size="${size}"
         % endif
         />
 
-      <p class="status" 
-         %if status != 'unsubmitted':
-         aria-hidden="true"
-         %endif
-         >
-          <span class="sr equation">
-              %if value:
-              ${value|h}
```

Comment:
Because screen reader will read the value in the text field and in the span as well. So here user just need to know whether the answer given is correct or not and not he value being inserted in text field.